charts/nginx-gateway-fabric/values.yaml

```diff
@@ -53,8 +53,8 @@ nginxGateway:
   image:
     # -- The NGINX Gateway Fabric image to use
     repository: ghcr.io/nginxinc/nginx-gateway-fabric
-    tag: edge
-    pullPolicy: Always
+    tag: 1.4.0
+    pullPolicy: IfNotPresent
```

Comment:
why do we change this to `IfNotPresent` for the release again?

Reply:
Always pulling the image is unnecessary network overhead if it already exists on the machine.

Really only beneficial for development when you are constantly rebuilding the same image tag and pushing.

Reply:
The overhead is just checking if there's a new digest available for the image, I feel like it's negligible and k8s says it's efficient.

Also with `IfNotPresent` users will not use the latest updated if we rebuild the images.

Reply:
Good points, though another point I see is that `Always` still requires auth, so that's part of the transaction. If auth is removed, then the pod won't start.

We at least allow a user to change this setting easily. I believe `IfNotPresent` is also the k8s default if you don't specify a policy.